gandalf.js

```diff
@@ -1,76 +1,79 @@
-const mergedArrays = [...array1, ...array2];
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?